docs/docs/monitoring/analytics/example-queries.mdx

```diff
@@ -0,0 +1,200 @@
+---
+id: example-queries
+sidebar_label: Example queries
+title: Example queries
+description:
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+This section aims to help you get started with ways on how to analyze the conversations your assistant is having.
+Example queries using SQL are provided, together with an example visualisation in Metabase.
+
+## Number of sessions per month
+
+A common high level usage metric of your assistant you could look at is the number of
+sessions per month. Here is how it would look:
+
+
+```sql
+SELECT
+  date_trunc('month', "public"."rasa_session"."timestamp") AS "first_seen",
+  count(*) AS "count"
+FROM "public"."rasa_session"
+GROUP BY 1
+ORDER BY 1 ASC
+```
+
+<figure align="center">
+  <img
+    alt="Number of sessions per month visualised in Metabase."
+    src={useBaseUrl("/img/analytics/graph-number-sessions-month.png")}
+    width="100%"
+  />
+  <figcaption>Number of sessions per month visualised in Metabase.</figcaption>
+</figure>
+
+## Number of sessions per channel
+
+If you're connecting your assistant to multiple channels, it could be useful to look at
+the number of sessions per channel, let's say per week. The query you would need for this metric is:
+
+```sql
+SELECT
+  "public"."rasa_user_message"."channel" AS "channel",
+  "public"."rasa_user_message"."timestamp" AS "timestamp",
+  count(distinct "public"."rasa_user_message"."session_id") AS "count"
+FROM "public"."rasa_user_message"
+GROUP BY 1, 2
+ORDER BY 1 ASC, 2 ASC
```

Comment:
```suggestion
SELECT
  "public"."rasa_sender"."channel" AS "channel",
  "public"."rasa_sender"."first_seen" AS "timestamp",
  count(distinct "public"."rasa_sender"."sender_key") AS "count"
FROM "public"."rasa_sender"
GROUP BY 1, 2
ORDER BY 1 ASC, 2 ASC
```

Reply:
this attribute moved from the message to the sender table

Reply:
and this is the number of "users" for this rather than sessions, sessions requires a join on the session table. what do we want to show, users per channel or sessions per channel @chandrikas ?

Reply:
users per channel is fine - I think that is a more understandable and universal concept. Sessions is a little more technical

Reply:
👍🏻